elements/lisk-transactions/src/10_delegate_transaction.ts

```diff
@@ -44,6 +44,29 @@ export const delegateRegistrationAssetSchema = {
 	},
 };
 
+const isNullCharacterIncluded = (input: string): boolean =>
+	new RegExp(/\0|\\u0000|\\x00/).test(input);
+
+const isUsername = (username: string): boolean => {
```

Comment:
we can remove from the `lisk-validator`?

Reply:
we discussed the removal part.. and we decided that ishan validator PR will do the cleanup.. i can remove if you still want to.